app/controllers/password_resets_controller.rb

```diff
@@ -0,0 +1,60 @@
+class PasswordResetsController < ApplicationController
+  before_action :get_user, only: %i(edit update)
```

Comment:
Bad naming. Don't use "get"/"set" with ruby. You can name it like as "load_user"